### DISCUSSION TITLE
Highest quality settings for 1080p60 streaming to a few people (both OvenMedia and OBS)?

### DISCUSSION THREAD

**Discussion by Gremious** [OvenMediaLabs/OvenMediaEngine#1255] [Help]
My use case is simply streaming video games to at most a handful of friends at a time, e.g. essentially an alternative to Discord screen sharing.

Currently I get incredibly low latency (awesome software) and *alright* quality. I play fighting games, so the screen is colorful and moving a lot, so my stream experiences "the bitrate crunch", getting all blocky, especially during action. In comparison, Discord nitro streaming was slightly, but consistently higher quality, so I'd like to achieve at least parity if possible?

Would like advise as to what are the highest quality setting I could tinker with for Oven Media/OBS/anything else, e.g. system?

---

Currently, I'm just streaming over WebRTC with all default settings, no transcoding or anything.
I don't believe I'm close to maxing out my server cpu or network or anything like that.

My OBS settings are as follows:

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/21365711/d2c8b092-f30a-4ada-bc34-3297afd585de)
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/21365711/6eb610ae-45ec-4c51-aff3-8985b4755f5b)

Server hardware:
```
CPU: Intel i3-6100U (4) @ 2.300GHz
GPU: Intel Skylake GT2 [HD Graphics 520]
Bandwidth: 150Mbps DL / 25Mbps UP
```

**Comment by naanlizard** [comment]
Your bitrate is too low for your encoder settings if you get artifacting

Your CPU isn't great though so I'm honestly surprised it's not skipping frames

You can try increasing your CPU usage preset too

**Comment by Gremious** [comment]
Ah I should have probably mentioned, the computer actually steaming/running obs is strong, has a good CPU, that's just the server hardware running OvenMedia that I noted down. I looked at htop, and I think server CPU didn't go beyond 15%. Haven't checked the PC though.

(Going to sleep now but) thanks, I will try higher bitrates tomorrow at least.

**Reply by naanlizard** [reply]
Try increasing your CPU usage preset then. Almost certainly that will fix it without just throwing more b bandwidth at the problem
